File: samples_generated_code/samples_3/qwen/baseline_code_java/code_row_65.java
import java.util.Scanner;

public class Main {
    public static int getValueFromArray(int[] array, int size, int index) {
        if (index < 0 || index >= size) {
            System.exit(1);
        }
        return array[index];
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int[] id_sequence = {10, 20, 30, 40, 50};
        int size = id_sequence.length;
        
        int index = scanner.nextInt();
        int result = getValueFromArray(id_sequence, size, index);
        System.exit(result);
    }
}

File: samples_generated_code/samples_3/phi4/baseline_code_py/code_row_65.py
import sys

def getValueFromArray(arr, size, idx):
    if 0 <= idx < size:
        return arr[idx]
    else:
        return -1

if __name__ == "__main__":
    id_sequence = [10, 20, 30, 40, 50]  # Example array
    try:
        index = int(input().strip())
        result = getValueFromArray(id_sequence, len(id_sequence), index)
        sys.exit(result)
    except ValueError:
        print("Invalid input")
        sys.exit(1)